apps/taxes/src/modules/avatax/avatax-connection.router.ts

```diff
@@ -28,7 +28,7 @@ const protectedWithConfigurationService = protectedClientProcedure.use(({ next,
       ...ctx,
       connectionService: new PublicAvataxConnectionService(
         ctx.apiClient,
-        ctx.appToken,
+        ctx.appId!,
```

Comment:
good example to:
1. Use value objects
2. Use object destructuring(name attributes) instead 
3. Test router